src/app/tools/color-palette/components/ColorPaletteGenerator.tsx

```
@@ -214,7 +273,9 @@ export default function ColorPaletteGenerator() {
 	useEffect(() => {
 		const saved = localStorage.getItem("color-palettes-v2");
 		if (saved) {
-			try { setSavedPalettes(JSON.parse(saved)); } catch (err) {}
+			try {
+				setSavedPalettes(JSON.parse(saved));
+			} catch (err) {}
```

Biome (2.4.15)

[error] This variable err is unused.

(lint/correctness/noUnusedVariables)